### PR TITLE
Improve local/initial configuration

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -10,9 +10,13 @@ set -euo pipefail
 # Default verbosity, common levels are 0,1,5,10
 verbosity="${verbosity:-1}"
 
-# Include stdlib from the lib directory
-# shellcheck disable=SC1091
-source lib/shtdlib/shtdlib.sh
+echo "Installing shtdlib"
```
Comment (hrpatel) [inline setup.sh:13]: @antevens Whats the best practice around downloading libs for repos? Should I download it into the local folder and source it? I can add the folder/script in `.gitignore` so it doesn't end up in the repo

Comment (antevens) [reply setup.sh:13]: I think it depends, some times submodules seem like the right approach but sometimes downloading on demand from raw github seems like a better approach.

File: bin/travis-ci/check_tag.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 set -eo pipefail
 
-# echo "Installing shtdlib"
+echo "Installing shtdlib"
```
Comment (antevens) [inline bin/travis-ci/check_tag.sh:20]: Perhaps we should check if shtdlib is already there or possibly even the version of it?

File: bin/travis-ci/check_tag.sh [hunk 3]
```diff
@@ -17,7 +17,7 @@
...
 shtdlib_local_path="/usr/local/bin/shtdlib.sh"
 sudo curl -s -L -o "${shtdlib_local_path}" https://github.com/sdelements/shtdlib/raw/master/shtdlib.sh
```
Comment (antevens) [inline bin/travis-ci/check_tag.sh:22]: Should we set the umask before doing this just to be safe, and then set it back?

**Review by antevens — DISMISSED** [review]

File: bin/travis-ci/check_tag.sh (hunk 2 above)
Comment (hrpatel) [reply bin/travis-ci/check_tag.sh:20]: This is in TravisCI so its a bare Ubuntu container/VM

File: bin/travis-ci/check_tag.sh (hunk 3 above)
Comment (hrpatel) [reply bin/travis-ci/check_tag.sh:22]: Even for CI jobs/scripts?

**Review by mroote — APPROVED** [review]
+1

**Comment by antevens** [comment]
+1

**Pull request merged**